src/routes/GroupDetail/GroupDetail.js

```diff
@@ -196,8 +200,9 @@ export class UnwrappedGroupDetail extends Component {
             </div>
           ) : (
             <div styleName='g.groupDescription'>
-              <GroupAboutVideoEmbed uri={group.aboutVideoUri} styleName='g.groupAboutVideo' />
```

Comment:
* Despite that this is moved out of the `div.groupDescription` it won't have a styling impact as there is no top margin or padding on `groupDescription`. There is a bottom margin on `aboutVideoUri` which will remain where it was before above the description, and now above the "enter a description" message (which is good).